docs/ref/models/meta.txt

```diff
@@ -144,6 +148,40 @@ Retrieving all field instances of a model
          <django.db.models.fields.related.ManyToManyField: groups>,
          <django.db.models.fields.related.ManyToManyField: user_permissions>)
 
+    This method may only be used once all models are loaded (that is, after a
+    call to :func:`django.setup()` completes); if called too soon, it will
+    raise an ``AppRegistryError``.
+
+.. _meta-local-fields:
+
+Working with a model before all apps are loaded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
`.. attribute:: Options.local_fields` (and use that for the link instead of adding `meta-local-fields`)